anago

```diff
@@ -1420,7 +1420,7 @@ else
   READ_RELEASE_BUCKETS+=("$RELEASE_BUCKET_USER" "$RELEASE_BUCKET_MOCK")
   GCRIO_REPO="${FLAGS_gcrio_repo:-kubernetes-release-test}"
```

Comment:
this probably needs to be `gcr.io/kubernetes-release-test` instead of just `kubernetes-release-test`

Reply:
done